lighty-examples/lighty-gnmi-community-restconf-app/src/main/java/io/lighty/examples/controllers/gnmi/Main.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021 PANTHEON.tech s.r.o. All Rights Reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v10.html
+ */
+
+package io.lighty.examples.controllers.gnmi;
+
+import io.lighty.applications.rcgnmi.app.RCgNMIApp;
+import java.io.IOException;
+
+public final class Main {
+    private static final String GNMI_CONFIGURATION = "/example_config.json";
+    private static final String TEST_CONFIG = "/src/test/resources/example_config.json";
+    private static final String PATH_TO_ASSEMBLY_RESOURCE = "/src/main/assembly/resources";
+    private static final String MODULE_NAME = "lighty-gnmi-community-restconf-app";
+    private static final String ASSEMBLY_RESOURCES
+            = "/lighty-examples/lighty-gnmi-community-restconf-app/src/main/assembly/resources";
+    private static final String LIGHTY = "lighty";
+    private static final String USER_DIR = "user.dir";
+
+    public static void main(String[] args) throws IOException {
+        final GnmiRestconfAppResources exampleResources = getGnmiRestconfAppResources();
+        GnmiSimulatorApp.main(new String[]{exampleResources.pathToExampleResources});
+        RCgNMIApp.main(new String[]{"-c", exampleResources.gnmiConfigPath});
+    }
```

Comment:
This start procedure should be refactored similar to https://github.com/PANTHEONtech/lighty/pull/752/commits/b8ba6a640469a95e04b06a0801bdb37a88051f31
The refactor will need the update/refactor of gNMI YANG modules loading